Makefile

```diff
@@ -57,6 +57,16 @@ deploy-policy-framework-hub:
 	kubectl apply -f https://raw.githubusercontent.com/open-cluster-management/governance-policy-propagator/main/deploy/operator.yaml -n $(KIND_HUB_NAMESPACE)
 
 
+deploy-community-policy-framework-hub:
```

Comment:
I would prefer us refactor `deploy-policy-framework-hub` to split into two make targets as this is almost identical to `deploy-policy-framework-hub` except the kind part.

Reply:
done as suggested.